inputs/values/dockers.json

```diff
@@ -13,12 +13,12 @@
   "samtools_cloud_docker": "us.gcr.io/broad-dsde-methods/gatk-sv/samtools-cloud:2022-06-10-v0.23-beta-9c6fbf56",
   "sv_base_docker": "us.gcr.io/broad-dsde-methods/gatk-sv/sv-base:2022-06-10-v0.23-beta-9c6fbf56",
   "sv_base_mini_docker": "us.gcr.io/broad-dsde-methods/gatk-sv/sv-base-mini:2022-06-10-v0.23-beta-9c6fbf56",
-  "sv_pipeline_base_docker": "us.gcr.io/broad-dsde-methods/gatk-sv/sv-pipeline:2023-01-19-v0.26.8-beta-5d1d5e0c",
-  "sv_pipeline_docker": "us.gcr.io/broad-dsde-methods/gatk-sv/sv-pipeline:2023-01-19-v0.26.8-beta-5d1d5e0c",
-  "sv_pipeline_hail_docker": "us.gcr.io/broad-dsde-methods/gatk-sv/sv-pipeline:2023-01-19-v0.26.8-beta-5d1d5e0c",
-  "sv_pipeline_updates_docker": "us.gcr.io/broad-dsde-methods/gatk-sv/sv-pipeline:2023-01-19-v0.26.8-beta-5d1d5e0c",
-  "sv_pipeline_qc_docker": "us.gcr.io/broad-dsde-methods/gatk-sv/sv-pipeline:2023-01-19-v0.26.8-beta-5d1d5e0c",
-  "sv_pipeline_rdtest_docker": "us.gcr.io/broad-dsde-methods/gatk-sv/sv-pipeline:2023-01-19-v0.26.8-beta-5d1d5e0c",
+  "sv_pipeline_base_docker": "sv-pipeline:cw-fix-refpanelgt-filter-c2bca8",
+  "sv_pipeline_docker": "sv-pipeline:cw-fix-refpanelgt-filter-c2bca8",
+  "sv_pipeline_hail_docker": "sv-pipeline:cw-fix-refpanelgt-filter-c2bca8",
+  "sv_pipeline_updates_docker": "sv-pipeline:cw-fix-refpanelgt-filter-c2bca8",
+  "sv_pipeline_qc_docker": "sv-pipeline:cw-fix-refpanelgt-filter-c2bca8",
+  "sv_pipeline_rdtest_docker": "sv-pipeline:cw-fix-refpanelgt-filter-c2bca8",
```

Comment:
Would this tag not mean the image should be pulled from the docker hub and not gcr?